### PR TITLE
Create minus.md

### PR DESCRIPTION
**Pull request by AlstrupJacob**
Create minus.md, solves #3812 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/datetime/terms/minus/minus.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  val t2 = t1.plus(125, DateTimeUnit.DAYS)
+
+  val p1 = t2.minus(t1, unit = DateTimeUnit.HOURS)
+  println(p1)
+
+  val t3 = t2.minus(p1)
+  println(t3)
+  println(t1)
+}
```
Comment (avdhoottt) [inline content/kotlin/concepts/datetime/terms/minus/minus.md:46]: This code example is not able to compile, Make sure the code executes properly.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@AlstrupJacob Please make these changes.

**Comment by avdhoottt** [comment]
@AlstrupJacob Don't commit the tags.md file as it is not a part of this entry.

**Comment by AlstrupJacob** [comment]
I cannot figure out why it wont pass, however the code example now compiles.

**Comment by SSwiniarski** [comment]
@KTom101 @caupolicandiaz again this is referring back to the Java libraries, not the `kotlinx.datetime` library. What is is we're looking for in the Kotlin/datetime concept?

**Review by SSwiniarski — APPROVED** [review]
@AlstrupJacob looks good.

**Comment by caupolicandiaz** [comment]
@avdhoottt @SSwiniarski I noticed that this entry overlooked much of the functionality of this method. So I updated the syntax description and example to better account for the options available. Please briefly review again to ensure no errors are present, since I've made significant edits.

**Comment by AlstrupJacob** [comment]
> @avdhoottt @SSwiniarski I noticed that this entry overlooked much of the functionality of this method. So I updated the syntax description and example to better account for the options available. Please briefly review again to ensure no errors are present, since I've made significant edits.

I wrote the documentation for the kotlinx version in the first place, but was told that was incorrect and to write it for the standard java time library instead.

**Comment by AlstrupJacob** [comment]
> > @avdhoottt @SSwiniarski I noticed that this entry overlooked much of the functionality of this method. So I updated the syntax description and example to better account for the options available. Please briefly review again to ensure no errors are present, since I've made significant edits.
> 
> I wrote the documentation for the kotlinx version in the first place, but was told that was incorrect and to write it for the standard java time library instead.

very confused as to which version of the minus function this documentation is supposedconcern

**Comment by AlstrupJacob** [comment]
whoops

**Comment by caupolicandiaz** [comment]
> > @avdhoottt @SSwiniarski I noticed that this entry overlooked much of the functionality of this method. So I updated the syntax description and example to better account for the options available. Please briefly review again to ensure no errors are present, since I've made significant edits.
> 
> I wrote the documentation for the kotlinx version in the first place, but was told that was incorrect and to write it for the standard java time library instead.

Apologies for the miscommunication. There was some confusion internally regarding these entries.

**Comment by AlstrupJacob** [comment]
> > > @avdhoottt @SSwiniarski I noticed that this entry overlooked much of the functionality of this method. So I updated the syntax description and example to better account for the options available. Please briefly review again to ensure no errors are present, since I've made significant edits.
> > 
> > 
> > I wrote the documentation for the kotlinx version in the first place, but was told that was incorrect and to write it for the standard java time library instead.
> 
> Apologies for the miscommunication. There was some confusion internally regarding these entries.

no worries just thought i would elaborate the current state of the pr

**Pull request merged**